File: Easy/Python/724. Find Pivot Index.py
class Solution(object):
    def pivotIndex(self, nums):
        pivot = -1
        left = 0
        total = 0

        for n in nums:
            total += n
        
        for i in range(len(nums)):
            if left * 2 == total - nums[i]:
                pivot = i
                break
            else:
                left += nums[i]
        
        return pivot

File: Easy/JavaScript/724. Find Pivot Index.js
var pivotIndex = function(nums) {
    let pivot = -1;
    let left = 0;
    let total = 0;

    for(let n of nums)
        total += n;
    
    for(let i = 0; i < nums.length; i++){
        if(left * 2 == total - nums[i]){
            pivot = i;
            break;
        }else{
            left += nums[i];
        }
    }

    return pivot;
};
